test/contracts/Staking/ExtraRewards.js

```diff
@@ -140,12 +141,15 @@ contract('StakingThales', accounts => {
         let StakingThales = await ethers.getContractFactory('StakingThales');
         let OngoingAirdrop = artifacts.require('OngoingAirdrop');
         let SNXRewards = artifacts.require('SNXRewards');
+        let AddressResolver = artifacts.require('AddressResolverHelper');
         let ThalesRoyale = artifacts.require('TestThalesRoyale');
         
         let OwnedUpgradeabilityProxy = artifacts.require('OwnedUpgradeabilityProxy');
         ThalesDeployed = await Thales.new({ from: owner.address });
         ThalesFeeDeployed = await Thales.new({ from: owner.address });
 		SNXRewardsDeployed = await SNXRewards.new();
```

Comment:
doesnt look formatted properly

Reply:
Also would be cool if you add the setAddressResolver as part of the upgrade script so that we dont forget :)

Reply:
need to update this before merge